build.sh

```diff
@@ -1,5 +1,13 @@
 #!/bin/bash
 set -euxo pipefail
+# Download the wasm-minimal-protocol repo and build wasi-stub tool from source.
+(
```

Comment:
i'm not really familar with shell script. will this run every time when you run ./install.sh?

Reply:
I'd suggest removing this part and move this to README if i understand it correctly

Reply:
I prefer move them to readme. In my experience i don't like random shell script installing things to my system --- I mean I prefer to read the code and copy-paste them to terminal.

Reply:
Sure thing! Just made those changes in the latest commit in this pull request 😉.